e2e/test/scenarios/admin-2/settings.cy.spec.js

```diff
@@ -275,24 +274,6 @@ H.describeWithSnowplow("scenarios > admin > settings", () => {
     cy.findByText(/Site URL/i);
   });
 
-  it(
-    "should display the order of the settings items consistently between OSS/EE versions (metabase#15441)",
-    { tags: "@OSS" },
-    () => {
-      IS_ENTERPRISE && H.setTokenFeatures("all");
-
-      const lastItem = !IS_ENTERPRISE ? "Cloud" : "Appearance";
-
-      cy.visit("/admin/settings/setup");
-      cy.findByTestId("admin-list-settings-items").within(() => {
-        cy.findAllByTestId("settings-sidebar-link").as("settingsOptions");
-        cy.get("@settingsOptions").first().contains("Setup");
-        // eslint-disable-next-line no-unsafe-element-filtering
-        cy.get("@settingsOptions").last().contains(lastItem);
-      });
-    },
-  );
-
```

Comment:
This should've been a unit test to begin with. It's no longer valid in a sense that "Appearance" now appears in the OSS version as well. The reason I didn't update this test wasn't because I'm lazy, but rather that I don't think it even makes sense. It's fragile to begin with.

But also, if it's tagged with `@OSS` how does it ever run against the enterprise version?
Anyway, good riddance, imo.